packages/commons/src/config/consumerServiceConfig.ts

```diff
@@ -12,3 +13,6 @@ export const ReadModelWriterConfig = KafkaConsumerConfig.and(ReadModelDbConfig);
 export type ReadModelWriterConfig = z.infer<typeof ReadModelWriterConfig>;
 export const readModelWriterConfig: () => ReadModelWriterConfig = () =>
   ReadModelWriterConfig.parse(process.env);
+
+export const emailManagerConfig: () => EmailManagerConfig = () =>
+  EmailManagerConfig.parse(process.env);
```

Comment:
Minor: just for naming consistency, I would move this into the `emailManagerConfig.ts` file, since not all consumers need it